selectionsort.py

```diff
@@ -0,0 +1,40 @@
+# Here we use dictionary data structure to store our count of values mapped to respective names in the above photos
+singers={"Selena Gomez": 112, "Taylor Swift": 178,"Ed Sheeran": 230,"Justin Bieber": 90,"John Legend": 190,"Beyoncé": 80}
```

Comment:
please write global variable in

> if __name__ == '__main__':

you can check in Jan's Binary Tree code.

Reply:
Will do and I searched for why is it necessary to add this, but nothing could clearly show me the reason. Could you please tell me why we should include this in ,sis?

Reply:
I updated if __name__ == '__main__':, but I wonder if the place I put in is correct